docs/src/documentation/05-development/04-migration-guides/v10.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: V10
+description: How to migrate to Orbit 10.0.0
+redirect_from:
+  - /migration-guides/v10/
+---
+
+# Orbit Migration Guide v10
+
+This migration guide focuses on the process of migrating from Orbit v9.1 to v10.0, as some breaking changes were introduced.
+With this guide we aim to walkthrough all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortless.
```

Comment:
```suggestion
With this guide, we aim to walk through all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortlessly.
```

Reply:
Changed on the others as well, as this is a common paragraph to all migration guides